source/install/upgrade_from_40.rst

```diff
@@ -34,7 +34,7 @@ Upgrading on Debian or Ubuntu
 |tip.run-all.root|
 
 1. Stop the ``mongod`` instance: :bash:`service mongod stop`
-#. Remove |PSMDB| |prev-version| packages: :bash:`apt-get remove percona-server-mongodb-36*`
+#. Remove |PSMDB| |prev-version| packages: :bash:`apt-get remove percona-server-mongodb-40*`
```

Comment:
4.0 packages do not have major version in their name (this change happened in 4.0 line), so `percona-server-mongodb-40*` should be `percona-server-mongodb*`